api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionIrsPaths.scala

```diff
@@ -28,18 +29,18 @@ trait SubmissionIrsPaths
   implicit val timeout: Timeout
 
   // institutions/<institutionId>/filings/<period>/submissions/<submissionId>/irs
-  // NOTE:  This is currently a mocked, static endpoint
   def submissionIrsPath(institutionId: String)(implicit ec: ExecutionContext) =
     path("filings" / Segment / "submissions" / IntNumber / "irs") { (period, submissionId) =>
```

Comment:
I noticed that the `submissionId` parameter is ignored (which makes sense... the endpoint can only return data for the latest submission, because of how the underlying table works). Should we consider removing that parameter from the endpoint and shortening the path? It could become `institutions/<id>/filings/<period>/irs`.

One argument for NOT removing that parameter: if we kept it, you could use the `RequestVerificationUtils` to evaluate requests and return a 404 when the institution/period/submission is nonexistent. Currently it returns a 500 Internal Server error for any institution or period not in the `lars_total_msa` table.